hello_world/cmake/Config.cmake.in

```diff
@@ -0,0 +1,6 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/HelloWorld-targets.cmake")
+
+include(CMakeFindDependencyMacro)
+find_dependency(CMakePublic)
```

Comment:
After finding it you still need to add the CMakePublic target to the imported HelloWorld target.